packages/opentelemetry-plugin-http/package.json

```diff
@@ -46,7 +46,6 @@
     "@types/shimmer": "^1.0.1",
     "@types/sinon": "^7.0.13",
     "@types/superagent": "^4.1.3",
-    "@opentelemetry/scope-base": "^0.0.1",
```

Comment:
it will re-added here:
https://github.com/open-telemetry/opentelemetry-js/pull/273/files#diff-b234751bed684f80529a7a50dbf4f52dR21

Reply:
You're right, added it back.